Action/Update.cs

```diff
@@ -25,10 +25,13 @@ public int RunCommand(CKAN.KSP ksp, object raw_options)
 
             user.RaiseMessage("Downloading updates...");
 
-            if (options.list_changes)
```

Comment:
Are we *sure* we want to be doing this? Before this change, `./ckan.exe update --list-changes --repo=https://github.com/KSP-CKAN/CKAN-meta/archive/master.zip` would work just fine. With this change, this now crashes the client.